docs/usage/config-overview.md

```diff
@@ -155,6 +155,17 @@ Inherited config may use all Repository config settings, and any Global config o
 
 For information on how the Mend Renovate App supports Inherited config, see the dedicated "Mend Renovate App Config" section toward the end of this page.
 
+#### Presets handling
+
+If the inherited config has presets, then Renovate:
+
+1. Resolves the presets
+1. Adds the resolved presets to the inherited config
+1. Merges the presets on top of the global config
+
+Using `ignorePresets` in your repository config to ignore certain presets will _not_ affect presets present in the inherited config.
```

Comment:
```suggestion
You can _not_ use `ignorePresets` in your repository config to ignore presets _within_ inherited config.
```

The current sentence is still hard to understand.

I think my suggestion is better. Please check for technical accuracy before applying! :wink: